app/app_stack.py

```diff
@@ -118,7 +119,15 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
                 "CDSAPI_URL": cds_url.string_value,
             },
         )
-
+        process_file.add_to_role_policy(
+            iam.PolicyStatement(
+                actions=[
+                    "states:ListStateMachines",
+                    "states:StartExecution",
+                ],
+                resources=["*"],
+            ),
+        )
```

Comment:
```suggestion
        )

```
och/eller ta bort kommentaren på nästa rad.